### PR TITLE
added readme

### PR DESCRIPTION
**Pull request by smitthakkar96**

### CODE REVIEW

**Comment by smitthakkar96** [comment]
Please consider reviewing this branch after we merge rest apis cause this branch also contains docs  for rest apis

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Fstat [![Build Status](https://travis-ci.org/gluster/fstat.svg?branch=master)](https://travis-ci.org/gluster/fstat)
+Fstat tracks the failures from build.gluster.org.
+
+## Installation
+- Before you start installation make sure you have working installation of python2
```
Comment (nigelbabu) [inline readme.md]: "a working installation"

Comment (nigelbabu) [inline readme.md]: Also add a step called "create a virtualenv"

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- In your virtualenv run ``` pip install -r requirements.txt ```
+- Hurray the installation is done
+
+## Using fstat
+- To serve the fstat app on localhost run ``` python manage.py runserver ```
```
Comment (nigelbabu) [inline readme.md]: This will most likely fail without configuring a database with `python manage.py db upgrade` at the start.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Use application.cfg to put all your development keys and debug flags
```
Comment (nigelbabu) [inline readme.md]: specify the flags without which nothing works.

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+## Consuming fstat data via rest apis
+
+###  ``` /api/failures ```
+This endpoint is used to get list of failures with the number of failure instances. The available filters for this endpoint are as follows:
+
+**Name**|**Description**
+:-----:|:-----:
+start\_date| The date from which you want to get the failures (format yyyy-mm-dd)
+end\_date| The date till which you want to get the failures (format yyyy-mm-dd)
+branch| filter for specific branch
```
Comment (nigelbabu) [inline readme.md]: Add an example using the filter.

File: readme.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+### ``` /api/failures/<fid> ```
+This endpoint will return the list of failure instances for a particular failureID, this failure id is specific to fstat. The available filters for this endpoint are as follows:
```
Comment (nigelbabu) [inline readme.md]: s/failureID/failure id/g

Comment (smitthakkar96) [inline readme.md]: @nigelbabu I didn't quite get you

File: readme.md (hunk 3 above)
Comment (smitthakkar96) [inline readme.md]: @nigelbabu can you please explain what are you trying to convey

**Pull request merged**